docs/content/preview/yugabyte-platform/troubleshoot/install-upgrade-yp-issues.md

```diff
@@ -64,3 +64,23 @@ do
   ssh $IP firewall-cmd --zone=public --add-port=6379/tcp;
 done
 ```
+
+## Configure load balancer for Helm charts
+
+If you experience issues with configuring a load balancer when installing and configuring YugabyteDB Anywhere on Kubernetes, you should define the Cross-Origin Resource Sharing (CORS) domain configuration by setting the [additionAllowedCorsOrigins](https://github.com/yugabyte/charts/blob/master/stable/yugaware/values.yaml#L66) value to the new domain involved.
+
+For Amazon Web Services (AWS) load balancers, the default service configuration is adequate; however, if customization is required, the backend protocol should be configured to HTTP by setting the following AWS load balancer controller annotations:
+
+- `aws-load-balancer-scheme` can be set to `internal` or `internet-facing` string value.
+- `aws-load-balancer-backend-protocol` and `aws-load-balancer-healthcheck-protocol` should be set to the `http` string value.
```

Comment:
If you are looking to customize the default AWS load balancer brought up in EKS by the YugabyteDB Anywhere helm chart, the following settings could help you customize the [AWS Load Balancer Controller](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.2/guide/service/annotations/) behavior involved in this setup.

Reply:
Changed